Source/WebCore/page/EventHandler.cpp

```diff
@@ -4969,7 +4968,7 @@ HandleUserInputEventResult EventHandler::handleTouchEvent(const PlatformTouchEve
     for (unsigned index = 0; index < points.size(); index++) {
         auto& point = points[index];
         PlatformTouchPoint::State pointState = point.state();
-        LayoutPoint pagePoint = documentPointForWindowPoint(frame, point.pos());
+        LayoutPoint pagePoint = LayoutPoint(frame->view()->windowToContents(point.pos()));
```

Comment:
You don't need to explicitly call the `LayoutPoint(...)` constructor.
```suggestion
        LayoutPoint pagePoint = frame->view()->windowToContents(point.pos());
```